caffe2/python/operator_test/layer_norm_op_test.py

```diff
@@ -344,7 +344,7 @@ def jit_layer_norm(
             expected_norm, expected_mean, expected_std = \
                 _layer_norm_with_affine_ref(axis, eps, X, gamma, beta)
             actual_norm, actual_mean, actual_std = jit_layer_norm(
-                torch.Tensor(X), torch.tensor(gamma), torch.tensor(beta),
+                torch.tensor(X), torch.tensor(gamma), torch.tensor(beta),
```

Comment:
Given that `X` is also a tensor, maybe it would be better to call `X.clone().detach()` (as the warning suggests). For consistency with the else case, I left it as `torch.tensor(X)`, though. Any thoughts?

Reply:
Actually, `torch.Tensor(X)` creates an alias to `X` (it does not clones its contents as `torch.tensor(X)`). But, looking at the _else_ part, it makes me believe that cloning was the intended behavior...

Reply:
+1 for using `clone().detach()`

Reply:
you never want to leave in code or docs that generate warnings

Reply:
I've just realized that `X` there is not a `Tensor`. The `X: torch.Tensor` we see above is a nested function: `jit_layer_norm`. Inside the actual function `test_layer_norm_op_jit`, `X` is a numpy array. So, I guess, `torch.tensor(X)` is the way to go.